_schemaV2.graphql

```diff
@@ -10814,6 +10814,33 @@ type PhoneNumberType {
   regionCode: String
 }
 
+type PreviewSavedSearch {
+  labels: [SearchCriteriaLabel]!
+}
+
+input PreviewSavedSearchAttributes {
+  acquireable: Boolean
+  additionalGeneIDs: [String]
+  artistIDs: [String]
+  artistNationalities: [String]
```

Comment:
**suggestion**: this filter is not yet supported in [SearchCriteriaAttributes](https://github.com/artsy/gravity/blob/main/app/graphql/types/inputs/search_criteria_attributes.rb#L3)

Reply:
Just noticed that too. About to push up an update which cleans up the order as well.